doc/Type/Any.pod6

```diff
@@ -781,6 +781,24 @@ Instead of throwing away the first C<$n> elements, it throws away everything
 I<but> the elements indicated by the WhateverCode, in this case all but the last
 one.
 
+=head2 method are
+
+Defined as:
+
+    multi method are()
+
+Return the strictest type (or role) to which B<all> elements of the list
```

Comment:
Ugh, I'm sorry to notice it late. I think most of the method descriptions use the "does" style of describing rather than "do", so it should be "returns" to be more consistent.
Otherwise looks good.